obsreport/obsreport_receiver.go

```diff
@@ -26,15 +26,23 @@ import (
 )
 
 const (
-	ReceiverKey  = "receiver"
+	// Key used to identify receivers in metrics and traces.
+	ReceiverKey = "receiver"
+	// Key used to identify the transport used to received the data.
 	TransportKey = "transport"
-	FormatKey    = "format"
+	// Key used to identify the format of the data received.
+	FormatKey = "format"
 
+	// Key used to identify spans accepted by the Collector.
 	AcceptedSpansKey = "accepted_spans"
-	RefusedSpansKey  = "refused_spans"
+	// Key used to identify spans refused (ie.: not ingested) by the Collector.
+	RefusedSpansKey = "refused_spans"
```

Comment:
It is better to have all the names prefixed with a namespace that ensures uniqueness. "otel.refused_spans"

Reply:
They are prefixed (by default) with `otel_col` so in Prometheus they look like:
```terminal
# HELP otelcol_receiver_accepted_metric_points Number of metric points successfully pushed into the pipeline.
# TYPE otelcol_receiver_accepted_metric_points counter
otelcol_receiver_accepted_metric_points{receiver="opencensus",transport="grpc"} 72
# HELP otelcol_receiver_accepted_spans Number of spans successfully pushed into the pipeline.
# TYPE otelcol_receiver_accepted_spans counter
otelcol_receiver_accepted_spans{receiver="jaeger",transport="collector_http"} 8749
otelcol_receiver_accepted_spans{receiver="opencensus",transport="grpc"} 305
otelcol_receiver_accepted_spans{receiver="zipkin",transport="http"} 8670
```
The prefix is done at the app level.